drivers/CoreRFIDReader/tests/CoreRFIDReader_test.cpp

```diff
@@ -0,0 +1,264 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+#include <functional>
+
+#include "CoreRFIDReader.h"
+#include "gtest/gtest.h"
+#include "mocks/leka/CoreBufferedSerial.h"
+#include "mocks/leka/EventQueue.h"
+
+using namespace leka;
+using namespace interface;
+
+using ::testing::Args;
+using ::testing::DoAll;
+using ::testing::ElementsAre;
+using ::testing::InSequence;
+using ::testing::Return;
+using ::testing::SaveArg;
+using ::testing::SetArrayArgument;
+
+class CoreRFIDReaderTest : public ::testing::Test
+{
+  protected:
+	CoreRFIDReaderTest() : reader(mockBufferedSerial) {};
+
+	// void SetUp() override {}
```

Comment:
Pour éviter de le refaire dans la plupart des tests

```suggestion
	void SetUp() override
	{
		reader.registerOnTagReadableCallback(callbackReadable.AsStdFunction());
		reader.registerOnTagDetectedCallback(callbackDetected.AsStdFunction());
		EXPECT_CALL(mockBufferedSerial, sigio).WillOnce(SaveArg<0>(&callbackSigio));
		reader.init();
	}
```